packages/react-router/lib/dom/ssr/routes-test-stub.tsx

```diff
@@ -91,7 +86,7 @@ export interface RoutesTestStubProps {
  */
 export function createRoutesStub(
   routes: StubRouteObject[],
-  context: AppLoadContext = {}
+  unstable_getContext?: () => unstable_InitialContext
```

Comment:
The typings are off, it used to take an `AppLoadContext`, now it takes a `() => unstable_InitialContext`. This means there's no way of using `createRoutesStub` with the old non-experimental API.

Migrating to the experimental API is not very desirable, even though I did in the end manage to do so, I'm worried that other developers might not commit to it.

It should either support both types or have an other API to support the old and the new use-case.